.github/scripts/trymerge.py

```diff
@@ -39,10 +37,21 @@
     get_revert_message,
 )
 
-class WorkflowCheckState(NamedTuple):
-    status: Optional[str]
-    url: str
-    name: str
+class JobCheckState:
+    def __init__(self, name: str, url: str, status: Optional[str], classification: Optional[str] = None):
+        self.name: str = name
+        self.url: str = url
+        self.status: Optional[str] = status
+        self.classification: Optional[str] = classification
+
+
+class WorkflowCheckState:
+    def __init__(self, name: str, url: str, status: Optional[str]):
+        self.name: str = name
+        self.url: str = url
+        self.status: Optional[str] = status
+        self.jobs: Dict[str, JobCheckState] = {}
```

Comment:
not sure which type you're referring to in this, is it the Dict[str, JobCheckState]? if yes, im open to names cuz if i do it it would just be "JobDict" which probably isnt very good

Reply:
Yeah, that was the one.

Maybe `JobStates` with the props: "jobName" and "state"